<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>880</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="fatRadioButton">
          <property name="text">
           <string>FAT</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="iNodeRadioButton">
          <property name="text">
           <string>iNode</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="plateGraphicInfoLabel">
      <property name="minimumSize">
       <size>
        <width>900</width>
        <height>250</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="freeLegendLabel">
        <property name="text">
         <string>Frei</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="freeLegendIconLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="usedLegendLabel">
        <property name="text">
         <string>Benutzt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="usedLegendIconLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="reservedLegendLabel">
        <property name="text">
         <string>Reserviert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="reservedLegendIconLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="notInitLegendLabel">
        <property name="text">
         <string>Nicht initialisiert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="notInitLegendIconLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="startUpButton">
        <property name="text">
         <string>Hochfahren</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reinitializeHDDButton">
        <property name="text">
         <string>HDD initialisieren</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="defragButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Defragmentieren</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createDirButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Ordner erstellen</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createFileButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Datei erstellen</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteSelectionButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Auswahl löschen</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QTreeView" name="fileSystemTreeView"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <bold>true</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="text">
             <string>Festplatteninformationen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="sizeInKbLabel">
            <property name="text">
             <string>Größe (in kB):         </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="blockSizeLabel">
            <property name="text">
             <string>Block Größe:                   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="blockAmountLabel">
            <property name="text">
             <string>Block Anzahl:                  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="freeBlocksLabel">
            <property name="text">
             <string>Freie Blöcke:                   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="freeDiskSpaceLabel">
            <property name="text">
             <string>Freier Speicher (in kB):   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="fragmentationLabel">
            <property name="text">
             <string>Fragmentierung:                </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <bold>true</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="text">
             <string>Dateiinformationen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="sizeInfoLabel">
            <property name="text">
             <string>Größe: Keine Datei ausgewählt</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="fileNameInfoLabel">
            <property name="text">
             <string>Name: Keine Datei ausgewählt</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pathInfoLabel">
            <property name="text">
             <string>Pfad:   Keine Datei ausgewählt</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1127</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
